modules/olm-operator-lifecycle-manager.adoc

```diff
@@ -179,38 +181,421 @@ required to update a CSV to the latest version in a channel (stepping through
 each intermediate version).
 
 [id='olm-architecture-operatorgroups-{context}']
-=== OperatorGroups
-
-The goal of _OperatorGroups_ is to bring multitenancy features to running
-Operators in a cluster in the easiest, most automated way.
-
-An OperatorGroup consists of a group of target namespaces as specified by the
-label selector in the spec, plus the namespace that the OperatorGroup is created
-within, known as the _Operator namespace_. The Operator namespace is always
-considered to also be a target namespace, without regard to matching the label
-selector. The Operator namespace is where the Operator actually runs and the
-target namespace(s) are namespaces within which the Operator has permissions to
-operate.
-
-After an OperatorGroup has been created, the focus is around the target
-namespaces and the resources contained in those namespaces. The status for an
-OperatorGroup is constantly updated to always list the namespaces matching the
-label selector as specified in the spec. In addition to maintaining an updated
-status, the OperatorGroup control loop also maintains creating RBAC rules and
-providing OperatorGroup knowledge to Operators. Some operations, such as copying
-CSVs and creating additional RBAC rules, are handled by the CSV control loop.
-
-RBAC rules are created to:
-
-* Restrict access to the API (CRDs) of the installed Operators. It is possible
-that the administrator does not want the full functionality of the Operator to
-be granted in all cases.
-* Give the Operator the necessary permissions to operate. This is tied to the
-specified OperatorGroup service account.
-
-OperatorGroup target namespace information is made available to Operators via
-annotations on the Deployment. Each CSV in the Operator namespace is copied into
-the target namespace(s), which is done in case a user does not have access to
-the Operator namespace. The copied CSV is annotated with the OperatorGroup name
-and the Operator namespace (the target namespace list is intentionally not
-included for security reasons).
+== OperatorGroups
+
+An _OperatorGroup_ is an OLM resource that provides multitenant configuration to
+OLM-installed Operators.
+
+An OperatorGroup selects a set of target namespaces in which to generate
+required RBAC access for its member Operators. The set of target namespaces is
+provided by a comma-delimited string stored in the `olm.targetNamespaces`
+annotation. This annotation is applied to member Operator's
+ClusterServiceVersion (CSV) instances and is projected into their deployments.
+
+[discrete]
```

Comment:
Given the scope of the current assembly, I think this is too much detail. I suggest moving the rest of this content to other modules that make up a "configuring an OperatorGroup" assembly. I think you need a task topic somewhere in here to describe how to make and load an OperatorGroup.